eclair-core/pom.xml

```diff
@@ -79,10 +79,10 @@
                 <activeByDefault>true</activeByDefault>
             </activation>
             <properties>
-                <bitcoind.url>https://bitcoin.org/bin/bitcoin-core-0.16.3/bitcoin-0.16.3-x86_64-linux-gnu.tar.gz
+                <bitcoind.url>https://bitcoin.org/bin/bitcoin-core-0.17.1/bitcoin-0.17.1-x86_64-linux-gnu.tar.gz
```

Comment:
Seems you haven't been starting from master branch?

Reply:
Ah my bad, i rebased this branch on `use-bitcoin-0.17.1` as there was a failing test (fixed in that branch) and i assumed that was going to master before the API revamp anyway,